services/apps/cache_worker/src/workflows/spawnDashboardCacheRefreshForAllTenants.ts

```diff
@@ -17,112 +16,84 @@ const activity = proxyActivities<typeof activities>({ startToCloseTimeout: '1 mi
 
 export async function spawnDashboardCacheRefreshForAllTenants(): Promise<void> {
   const tenants = await activity.getAllTenants()
-  const segmentsEnabled = await activity.isSegmentsEnabled()
   const info = workflowInfo()
 
-  if (segmentsEnabled) {
-    // we should spawn refreshDashboardCache for each tenant-segment couples
+  const SEGMENT_PAGE_SIZE = 250
 
-    const SEGMENT_PAGE_SIZE = 250
+  for (const tenant of tenants) {
```

Comment:
can u run `npm run format` for this service?

Reply:
done